examples/google_hangouts_bot.js

```diff
@@ -0,0 +1,95 @@
+var Botkit = require('../lib/Botkit.js');
+
+var controller = Botkit.googlehangoutsbot({
+    endpoint: 'Axjn86rTGRQwisaYFyT0XZyiOCh7rZUPGx1A',
+    token: "YOUR_TOKEN",
+    debug: true,
+});
+
+var bot = controller.spawn({});
+
+controller.setupWebserver(3000, function (err, webserver) {
+    controller.createWebhookEndpoints(webserver, bot, function () {
+        console.log(`🚀 Congratulation, the web server is online!`);
+    });
+});
+
+controller.on('message_received', function (bot, message) {
+    bot.reply(message, `You said '${message.text}'`);
+});
+
+controller.hears('new thread', 'message_received', function (bot, message) {
+    bot.replyAsNewThread(message, `Hello ! this is a new thread`);
+});
+
+controller.hears('new thread', 'message_received', function (bot, message) {
+    bot.replyAsNewThread(message, `Hello ! this is a new thread`);
+});
+
```

Comment:
Isn't that a duplicate handler for `new thread` ?

Reply:
it is ! nice catch @arnaudjolly